oe_corporate_site_info.install

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate Site Information installation file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_corporate_site_info_install(): void {
+  // Configure the RDF SKOS graphs.
+  if (!\Drupal::service('config.installer')->isSyncing()) {
```

Comment:
Add a @todo here to remove this and update the hook_install signature which in 8.9 receives the `$is_syncing` parameter: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_install/8.9.x